deploy/local/live-csc/.env

```diff
@@ -7,35 +7,42 @@ DOCKERFILE_PATH_COMMANDER=../../../../LOVE-commander/
 
 # INTERACTION BETWEEN SERVICES
 SERVER_URL=*
-LOVE_PRODUCER_WEBSOCKET_HOST=love-nginx-mount/manager/ws/subscription
+#LOVE_PRODUCER_WEBSOCKET_HOST=love-nginx-mount/manager/ws/subscription
```

Comment:
In general we prefer not to have commented out lines in our code and there suddenly are many in this file. Are these absolutely necessary? If not, please remove them.